src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGenerator.cs

```diff
@@ -615,6 +623,17 @@ private OpenApiMediaType CreateResponseMediaType(ModelMetadata modelMetadata, Sc
             };
         }
 
+        private bool IsFromFormAttributeUsedWithIFormFile(ApiParameterDescription apiParameter)
```

Comment:
Ths method doesn't appear to use any state from the class, so can be static:

```suggestion
        private static bool IsFromFormAttributeUsedWithIFormFile(ApiParameterDescription apiParameter)
```

Reply:
Done